### DISCUSSION TITLE
Breaking API Changes? Please notify me!

### DISCUSSION THREAD

**Discussion by admin** [Traewelling/traewelling#2619] [Announcements]
Dear API Consumers,

Please be aware that our API is still undergoing regular changes. In the event of breaking changes, we can tag you in the corresponding pull request (PR) to keep you informed.

To receive these notifications, please write a message in this thread indicating that you would like to join the notification team. We will then send you an invitation, which will be valid for 7 days. Once you have been invited, we will hide your comment from this thread for a better overview. You can then also delete it if you want.

If you no longer wish to receive notifications, simply leave the team.

Please also check [API_CHANGELOG.md](https://github.com/Traewelling/traewelling/blob/develop/API_CHANGELOG.md) regularly.

**Comment by sanojkl** [comment]
please invite me :)

**Comment by lambda-crime** [comment]
könnte ich 1 invite kriegen? :point_right::point_left:

**Comment by shedowe19** [comment]
Moin ^^ könnte ich 1 invite kriegen?

**Comment by scaryMikrowelle** [comment]
pls notify me, thank youu
